betterdgg/modules/mentions.js

```diff
@@ -7,9 +7,12 @@
                 gui.lines.on('mousedown', 'div.user-msg a.user', function() {
                     if (bdgg.settings.get('bdgg_highlight_selected_mentions')) {
                         var username = $(this).text();
-                        gui.lines.find(':contains(' + username + ')').addClass('focused');
+                        var users = [$(this).parents().parents().find(".chat-user:contains(" + username + ")")];
+                        for (var i = 0; i < $(users)[0].length - 1; i++) {
+                            window.destiny.chat.gui.stylesheet.insertRule('.user-msg[data-username="' + ($($(users)[0].parent().parent()[i]).find(".user").text().toLowerCase()) + '"]{opacity:1 !important;}', window.destiny.chat.gui.stylesheet.cssRules.length);
```

Comment:
Furthermore, we have defined `gui = window.destiny.chat.gui` above so you should use that alias here.